10x/count/count.wdl

```diff
@@ -148,20 +172,24 @@ task chunk_reads_main {
 
   output {
     File outs = "_outs"
-    Array[File] fastq_chunks = glob("files/*.fastq")
+    Array[File] fastq_chunks = glob("fastq_chunk_*.tar.gz")
   }
 }
 
 task chunk_reads_join {
   # Array of outs objects from the chunk_reads_main steps. All this step is doing
   # is joining some dicts together.
   Array[File] outs
+  Array[Array[File]] fastq_chunks
```

Comment:
I think for maintainability we should put some comments around inputs that aren't clear from their name/type that describe what they are. What you've got on lines 149 + 150 is good; we could copy it here. What do you think?

Reply:
Yeah good idea. I added a little note about what the `fastq_chunks` are supposed to be.